bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/PredefinedTypes.java

```diff
@@ -140,6 +140,10 @@
                                                          new Module(BALLERINA_BUILTIN_PKG_PREFIX, XML_LANG_LIB, null),
                                                          TypeTags.XML_TEXT_TAG, true);
 
+    public static final XMLType TYPE_TEXT_NEVER = new BXMLType(TypeConstants.XML_TEXT,
```

Comment:
How about 
```suggestion
    public static final XMLType TYPE_XML_NEVER = new BXMLType(TypeConstants.XML_TEXT,
```
instead?